src/components/nodes/callActivity/callActivity.vue

```diff
@@ -39,9 +41,10 @@ export default {
       const { height } = this.shape.size();
 
       if (labelHeight + labelPadding !== height) {
-        const newHeight = Math.max(labelHeight + 15, taskHeight);
+        const newHeight = Math.max(labelHeight + 15 + 2 * MARKER_SIZE, taskHeight);
```

Comment:
It looks like 15 was a magic number that should have been cleaned up, but now there is another magic number being added (2) and this whole expression is now harder to understand.

It seems that 15 should be a reference to `labelPadding`, and the expressions should be grouped for better readability:
```js
(labelHeight + labelPadding) + (MARKER_SIZE * 2)
```

Once you do that, you'll see that `labelHeight + labelPadding` is used above as well, and you could assign it a meaningful variable name.

However, this code looks very similar to code in `textAnnotation.vue`; take a look at the `updateNodeText` method in that component.